pkg/fileutil/util.go

```diff
@@ -104,3 +106,19 @@ func StartLoadDynamicFile(filename string, callBack FileChangeCallBack, stopCh <
 		}
 	}, time.Second, stopCh)
 }
+
+func CalculateTimeDeltaFromUnixInSeconds(from string, to int64) (float64, error) {
```

Comment:
We are converting the data type from int64 to string and converting it back to int64 for comparison.
Please make it more focused by passing (string, int64) and only converting the first argument to int64.

Reply:
change the param names to startTime and endTime

Reply:
can we drop the to parameter completely? The callers are passing time.Now anyway.